tests/ctest_ut/ctestunittests.c

```diff
@@ -79,5 +79,14 @@ int main()
         }
     }
 
+    {
+        size_t temp_failed_tests = 0;
+        CTEST_RUN_TEST_SUITE(WindowsTypesTests, temp_failed_tests);
+        if (temp_failed_tests != 2) // 2 expected failing tests
```

Comment:
>2) // 2 [](start = 33, length = 7)

where does 2 come from? I count 4 failing tests

Reply:
They are 5 indeed. It was 2 when there were only 2 types. But then I added the rest and now they are 5. That's why my builds were failing :-).

---
In reply to: [309833697](https://github.com/Azure/azure-ctest/pull/34#discussion_r309833697) [](ancestors = 309833697)